### PR TITLE
Workspace auditing workbook

### PR DESCRIPTION
**Pull request by sarah-yo**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Workbooks/WorkbooksMetadata.json [hunk 1]
```diff
@@ -944,5 +944,18 @@
     "templateRelativePath": "NormalizedNetworkEvents.json",
     "subtitle": "",
     "provider": "Microsoft"
+  },
+     {
+  "workbookKey": "WorkspaceAuditingWorkbook",
+  "logoFileName": "paloalto_logo.svg",
```
Comment (Amitbergman) [inline Workbooks/WorkbooksMetadata.json:950]: Is this correct? Same logo as paloalto?

File: Workbooks/WorkbooksMetadata.json [hunk 2]
```diff
@@ -944,5 +944,18 @@
...
+  "description": "Workspace auditing report\r\nUse this report to understand query runs across your workspace.",
+  "dataTypesDependencies": [ "LAQueryLogs" ],
+  "dataConnectorsDependencies": [ "" ],
+  "previewImagesFileNames": [ "WorkspaceAuditgingWhite.png", "WorkspaceAuditingBlack.png" ],
+  "version": "1.0",
+  "title": "Workspace audit",
```
Comment (Amitbergman) [inline Workbooks/WorkbooksMetadata.json:956]: Maybe "query auditing"? Or it is indeed audit on the workspace?

File: Workbooks/WorkbooksMetadata.json [hunk 3]
```diff
@@ -944,5 +944,18 @@
...
     "subtitle": "",
     "provider": "Microsoft"
+  },
+     {
+  "workbookKey": "WorkspaceAuditingWorkbook",
+  "logoFileName": "paloalto_logo.svg",
+  "description": "Workspace auditing report\r\nUse this report to understand query runs across your workspace.",
+  "dataTypesDependencies": [ "LAQueryLogs" ],
+  "dataConnectorsDependencies": [ "" ],
+  "previewImagesFileNames": [ "WorkspaceAuditgingWhite.png", "WorkspaceAuditingBlack.png" ],
```
Comment (Amitbergman) [inline Workbooks/WorkbooksMetadata.json:954]: typo in auditing, you wrote audit**g**ing

File: Workbooks/WorkbooksMetadata.json (hunk 1 above)
Comment (sarah-yo) [reply Workbooks/WorkbooksMetadata.json]: Fixed

File: Workbooks/WorkbooksMetadata.json (hunk 3 above)
Comment (sarah-yo) [reply Workbooks/WorkbooksMetadata.json]: fixed

File: Workbooks/WorkbooksMetadata.json (hunk 2 above)
Comment (sarah-yo) [reply Workbooks/WorkbooksMetadata.json:956]: It's more than just query auditing so would rather leave the name as is

**Review by Amitbergman — CHANGES_REQUESTED** [review]
Just fix the typo in the file name :)

**Review by Amitbergman — APPROVED** [review]

**Pull request merged**